dora/core/client/fs/src/main/java/alluxio/client/file/DoraCacheFileSystem.java

```diff
@@ -275,10 +275,8 @@ public FileOutStream createFile(AlluxioURI alluxioPath, CreateFilePOptions optio
       outStreamOptions.setMountId(status.getMountId());
       outStreamOptions.setAcl(status.getAcl());
 
-      FileOutStream ufsOutStream = mDelegatedFileSystem.createFile(ufsFullPath, options);
```

Comment:
Is it possible to keep this on client? we can have a flag to switch on and off write through worker

Reply:
Yes. Please see Line 283 - 288.
There is still a fallback to UFS, and there is a switch to enable/disable such fallback.